src/frontend/src/hooks/finance.hooks.ts

```diff
@@ -40,11 +40,11 @@ import {
 
 export interface CreateReimbursementRequestPayload {
   vendorId: string;
-  account: ClubAccount;
   dateOfExpense: Date;
   expenseTypeId: string;
   reimbursementProducts: ReimbursementProductCreateArgs[];
   totalCost: number;
+  account?: ClubAccount;
```

Comment:
Why

Reply:
yeahhh, I don't like this, it was so we don't have to set a default value for refund source (bc it could be an invalid value if the associated account code doesn't allow it). I'll see if I can figure out a way not to do this